createEditUrl.js

```diff
@@ -0,0 +1,26 @@
+/**  list of resources that does not need to be edited */
+const UNEDITABLE_RESOURCES = [ 
+  'release-notes',
```

Comment:
why not the release notes?

Reply:
Hi @fredmaggiowski I thought that the release notes were generated automatically by a changelog. If this is not the case, I can remove it